torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2700,6 +2700,46 @@ def sample_inputs_fliplr_flipud(op_info, device, dtype, requires_grad, **kwargs)
     )
     return [SampleInput(tensor) for tensor in tensors]
 
+def sample_inputs_fmod_remainder(op_info, device, dtype, requires_grad, *, autodiffed_args=False, **kwargs):
```

Comment:
What would it take to combine this function with the current `sample_inputs_binary_pwise`?

https://github.com/pytorch/pytorch/blob/71f4c5c1f436258adc303b710efb3f41b2d50c4e/torch/testing/_internal/common_methods_invocations.py#L683

Reply:
Not that I'm suggesting we do that, necessarily, but it would be interesting to try and understand. My guess is that it's a little too tricky because of the autodiffing.

Reply:
I agree, we can use this function `sample_inputs_binary_pwise`, but it might increase the complexity of the function (in terms of readability) as we might just add `autodiff` arg to the signature, and create tensors separately for that case.

Reply:
Agreed. Keeping them separate sounds good (at least for now)